### DISCUSSION TITLE
Viruses in the download package

### DISCUSSION THREAD

**Discussion by DaitinYT** [lllyasviel/Fooocus#3394] [Q&A]
I have downloaded the package in zip format, I have extracted it and before running it, it scans it and it shows that it has a virus called PUA:Win32/Vigua.A When I see that it does not run the program, can someone tell me something about it or if it is a fake? positive thanks.

**Comment by mashb1t** [comment]
There are no viruses in the zip file, i've also scanned it with Bitdefender and Eset. Where did you download the file from and which Antivirus software do you use?

**Reply by DaitinYT** [reply]
Hello, thank you for the prompt response, I downloaded it from here https://github.com/lllyasviel/Fooocus in the download section my Windows Defender marked that virus before installing I scanned it and it skipped that

**Reply by mashb1t** [reply]
PUAs are not viruses, they are "potential unwanted applications", which i assume the Gradio share feature has been marked as as it published a port to the internet when Fooocus is used with `--share`.

Which exact file has been identified containing the warning?

**Reply by DaitinYT** [reply]
The procedure I do is to right click on the zip folder and when the defender finishes its scan it says that it finds that virus but does not specify a route, my fear is losing my accounts, I have already been hacked twice and now the truth is I'm afraid
